conf.py

```diff
@@ -70,7 +70,10 @@
     'images',
     'book/cartridge/cartridge_overview.rst',
     'book/cartridge/CONTRIBUTING.rst',
+    'book/cartridge/topics',
+    'book/cartridge/cartridge_api/modules/cartridge.test-helpers.rst',
```

Comment:
What is wrong with this page? https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge.test-helpers/

Reply:
this page represents only 'see also' lines to other pages (cartridge.test-helpers.server, cartridge.test-helpers.cluster, cartridge.test-helpers.ectd, cartridge.test-helpers.stateboard) but doesn't really links to them. We have these pages in table of contents so cartridge.test-helpers is redundant here.

Reply:
But it causes several lines of warnings. By removing it, I reduced them to 1